modular_robot_simulation/revolve2/modular_robot_simulation/build_multi_body_systems/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Builders for simulating modular robots."""
+from ._body_to_multi_body_system_converter import BodyToMultiBodySystemConverter
+from ._builder import Builder
+
+__all__ = ["BodyToMultiBodySystemConverter", "Builder"]
```

Comment:
I don't think Builder is used anywhere outside of this module so let's not expose it.